ext/RMagick/rmutil.c

```diff
@@ -1781,6 +1781,38 @@ rm_ensure_result(Image *image)
 }
 
 
+/**
+ * Checks if opacity_or_alpha is a named argument called alpha and returns the alpha value or
+ * converts the unnamed opacity value to alpha.
+ *
+ * No Ruby usage (internal function)
+ *
+ * @opacity_or_alpha an opacity or a named alpha value
+ * @method_name the name of the method
+ * @argument_name the name of the argument
+ */
+Quantum
+rm_get_named_alpha_value(VALUE opacity_or_alpha, const char *method_name, const char *argument_name)
```

Comment:
Why do we need to pass in `argument_name` here instead of putting `"alpha"` inline below?

Reply:
See my comment below.